lib/Sema/CSDiagnostics.cpp

```diff
@@ -1477,3 +1477,223 @@ bool MissingMemberFailure::diagnoseAsError() {
   corrections.noteAllCandidates();
   return true;
 }
+
+bool AllowTypeOrInstanceMemberFailure::diagnoseAsError() {
```

Comment:
Could you please run clang-format on this method again? It seems incorrectly indented...